packages/core/src/services/ai/index.ts

```diff
@@ -126,6 +128,7 @@ export async function ai(
     onFinish: (event) => {
       providerLogHandler({
         uuid: uuidv4(),
+        generatedAt: logGeneratedAt,
```

Comment:
Introduced `generatedAt` to avoid depending on `createdAt`. I already had issues on CI where `createdAt` was identical and test was failing.